bokeh/charts/chart.py

```diff
@@ -117,19 +118,6 @@ class Chart(Plot):
     Height of the rendered chart, in pixels.
     """)
 
-    filename = Either(Bool(False), String, help="""
-    A name for the file to save this chart to.
-    """)
-
-    server = Either(Bool(False), String, help="""
-    A name to use to save this chart to on server.
-    """)
-
-    notebook = Either(Bool(False), String, help="""
-    Whether to display the plot inline in an IPython/Jupyter
-    notebook.
```

Comment:
we do have the notion of a deprecated property now, we might want to use that here to warn people. @canavandl can you point @fpliger at the deprecated property code/example?